hashicorp/boundary-ui
# chore: 🤖 Update babel-loader

## Description

Update babel-loader

### How to test:
Building the 3 projects (admin, desktop and rose) that depend on it individually and validate the build was successful.
As a sanity check, also run the UI's of those 3 projects.
